### PR TITLE
More editing work on TRPL

### PR DESCRIPTION
**Pull request by steveklabnik**
Fill out blank section headers. Copy edit the entire first section.

### CODE REVIEW

File: src/doc/trpl/hello-world.md [hunk 1]
```diff
@@ -147,15 +147,15 @@ This prints out our `Hello, world!` text to our terminal.
...
+give someone a `.rb` or `.py` or `.js` file, they need to have a
 Ruby/Python/JavaScript implementation installed, but you just need one command
-to both compile and run your program. Everything is a tradeoff in language design,
-and Rust has made its choice.
+to both compile and run your program. Everything is a tradeoff in language
+design, and Rust has made its choice.
 
 Congratulations! You have officially written a Rust program. That makes you a
-Rust programmer! Welcome.
+Rust programmer! Welcome. 👍
```
Comment (nagisa) [inline src/doc/trpl/hello-world.md]: 🎊🎉👏

File: src/doc/trpl/hello-cargo.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
-Rust projects. Cargo is currently in a pre-1.0 state, just like Rust, and so it
-is still a work in progress. However, it is already good enough to use for many
-Rust projects, and so it is assumed that Rust projects will use Cargo from the
-beginning.
+[Cargo][cratesio] is a tool that Rustaceans use to help manage their Rust
+projects. Cargo is currently in a pre-1.0 state, and so it is still a work in
+progress. However, it is already good enough to use for many Rust projects, and
+so it is assumed that Rust projects will use Cargo from the beginning.
+
+[cratesio]: https://crates.io
```
Comment (tshepang) [inline src/doc/trpl/hello-cargo.md]: Maybe point to the documentation instead... http://doc.crates.io.

File: src/doc/trpl/hello-cargo.md [hunk 3]
```diff
@@ -83,13 +76,33 @@ Hello, world!
...
+```bash
+$ cargo build
+   Compiling hello_world v0.0.1 (file:///home/yourname/projects/hello_world)
+     Running `target/debug/hello_world`
+Hello, world!
+```
+
+This hasn’t bought us a whole lot over our simple use of `rustc`, but think
+about the future: when our project has more than one file, we would need to
+call `rustc` more than once and pass it a bunch of options to tell it to build
```
Comment (huonw) [inline src/doc/trpl/hello-cargo.md]: `rustc` will load files as necessary for `mod`s automatically (i.e. not like  c/c++ compiler that makes per-file object files etc), so this statement is possibly not what you're trying to convey.

File: src/doc/trpl/hello-cargo.md [hunk 4]
```diff
@@ -160,9 +180,20 @@ fn main() {
...
-Now that you've got the tools down, let's actually learn more about the Rust
+[guide]: http://doc.crates.io/guide.html
+
+Now that you’ve got the tools down, let’s actually learn more about the Rust
 language itself. These are the basics that will serve you well through the rest
 of your time with Rust.
+
+You have two options: Dive into a project with ‘[Learn Rust][learnrust]’, or
+start from the bottom and work your way up with ‘[Syntax and
+Semantics][synatax]’. More experienced systems programmers will probably prefer
```
Comment (huonw) [inline src/doc/trpl/hello-cargo.md]: s/synatax/syntax/

File: src/doc/trpl/hello-world.md (hunk 1 above)
Comment (steveklabnik) [inline src/doc/trpl/hello-world.md]: yeah, maybe i should do that :)

File: src/doc/trpl/hello-cargo.md (hunk 3 above)
Comment (steveklabnik) [inline src/doc/trpl/hello-cargo.md]: I was thinking multi-crate stuff with dependencies. Thoughts on how to allude to that without saying it directly?

File: src/doc/trpl/hello-cargo.md (hunk 2 above)
Comment (steveklabnik) [inline src/doc/trpl/hello-cargo.md]: I thought about it, but wasn't sure. Probably better, yeah

File: src/doc/trpl/hello-cargo.md (hunk 3 above)
Comment (steveklabnik) [inline src/doc/trpl/hello-cargo.md]: I'll just go with 'more complex'

**Comment by steveklabnik** [comment]
Updated!

**Comment by steveklabnik** [comment]
Same here, to fix issues like #24252, I'm going to self-r+ these, if there's other nits, send an issue or PR :heart:

**Pull request merged**